contracts/Uniswap/UniswapConfig.sol

```diff
@@ -711,33 +777,377 @@ contract UniswapConfig {
+        if (i == 27) {
+            underlying = underlying27;
+            symbolHash = symbolHash27;
+            baseUnit = baseUnit27;
+            priceSource = priceSource27;
+            fixedPrice = fixedPrice27;
+            uniswapMarket = uniswapMarket27;
+            reporter = reporter27;
+            reporterMultiplier = reporterMultiplier27;
+        }
+        if (i == 28) {
+            underlying = underlying28;
+            symbolHash = symbolHash28;
+            baseUnit = baseUnit28;
+            priceSource = priceSource28;
+            fixedPrice = fixedPrice28;
+            uniswapMarket = uniswapMarket28;
+            reporter = reporter28;
+            reporterMultiplier = reporterMultiplier28;
+        }
+        if (i == 29) {
+            underlying = underlying29;
+            symbolHash = symbolHash29;
+            baseUnit = baseUnit29;
+            priceSource = priceSource29;
+            fixedPrice = fixedPrice29;
+            uniswapMarket = uniswapMarket29;
+            reporter = reporter29;
+            reporterMultiplier = reporterMultiplier29;
+        }
+        if (i == 30) {
+            underlying = underlying30;
+            symbolHash = symbolHash30;
+            baseUnit = baseUnit30;
+            priceSource = priceSource30;
+            fixedPrice = fixedPrice30;
+            uniswapMarket = uniswapMarket30;
+            reporter = reporter30;
+            reporterMultiplier = reporterMultiplier30;
+        }
+        if (i == 31) {
+            underlying = underlying31;
+            symbolHash = symbolHash31;
+            baseUnit = baseUnit31;
+            priceSource = priceSource31;
+            fixedPrice = fixedPrice31;
+            uniswapMarket = uniswapMarket31;
+            reporter = reporter31;
+            reporterMultiplier = reporterMultiplier31;
+        }
+        if (i == 32) {
+            underlying = underlying32;
+            symbolHash = symbolHash32;
+            baseUnit = baseUnit32;
+            priceSource = priceSource32;
+            fixedPrice = fixedPrice32;
+            uniswapMarket = uniswapMarket32;
+            reporter = reporter32;
+            reporterMultiplier = reporterMultiplier32;
+        }
+        if (i == 33) {
+            underlying = underlying33;
+            symbolHash = symbolHash33;
+            baseUnit = baseUnit33;
+            priceSource = priceSource33;
+            fixedPrice = fixedPrice33;
+            uniswapMarket = uniswapMarket33;
+            reporter = reporter33;
+            reporterMultiplier = reporterMultiplier33;
+        }
+        if (i == 34) {
+            underlying = underlying34;
+            symbolHash = symbolHash34;
+            baseUnit = baseUnit34;
+            priceSource = priceSource34;
+            fixedPrice = fixedPrice34;
+            uniswapMarket = uniswapMarket34;
+            reporter = reporter34;
+            reporterMultiplier = reporterMultiplier34;
+        }
+
+        return TokenConfig({
```

Comment:
Does this save contract space?

Reply:
Indeed it does free up quite a few storage slots! This change had the most impact on saving contract space.